osv/test_tools/test_repository.py

```diff
@@ -0,0 +1,158 @@
+"""test_repository"""
+import pygit2
+import json
+from datetime import datetime
+from enum import Enum
+import os
+import shutil
+import uuid
+
+
+class TestRepository:
+  """ Utilitary class to create a test repository for the git tests
```

Comment:
Fix spelling mistake

```suggestion
  """ Utility class to create a test repository for the git tests
```